torch/lib/c10d/ProcessGroup.cpp

```diff
@@ -55,8 +55,11 @@ bool ProcessGroup::Work::wait(std::chrono::milliseconds timeout) {
   return true;
 }
 
-void ProcessGroup::Work::abort() {
-  TORCH_CHECK(false, "ProcessGroup::Work::abort not implemented.")
+void ProcessGroup::Work::abort(){
+    TORCH_CHECK(false, "ProcessGroup::Work::abort not implemented.")}
```

Comment:
wrong indent and spacing, please use clang-format to format code.

Reply:
That was done by VS code. Actually, when I do
```
void ProcessGroup::Work::abort(){
  TORCH_CHECK(false, "ProcessGroup::Work::abort not implemented.")}
```
VS Code gives `clang-format` warning. Maybe it is a bug with VS code's language service.

Reply:
what about the following?

```c++
void ProcessGroup::Work::abort() {
  TORCH_CHECK(false, "ProcessGroup::Work::abort not implemented.");
}
```

Reply:
That worked, thanks!